.github/workflows/gitlab.yml

```diff
@@ -46,10 +49,9 @@ jobs:
         run: |
             go mod download
 
-      - name: Run GitLab PAT E2E  #using retry because the GitHub token is being throttled.
+      - name: Run GitLab PAT E2E  # skip if auth token is not available
+        if: ${{ env.GITLAB_AUTH_TOKEN != '' }}
```

Comment:
since this is related to a dependabot feature, would it make more sense to check for `github.actor != "dependabot[bot]"`
https://github.blog/changelog/2021-02-19-github-actions-workflows-triggered-by-dependabot-prs-will-run-with-read-only-permissions/